extensions/projection/README.md

```diff
@@ -0,0 +1,75 @@
+# Projection Extension Specification (`proj`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+This document explains the fields of the STAC Projection (`proj`) Extension to a STAC Item. Here `proj` is short
+for "projection", and not a reference to the use of the PROJ/PROJ4 formats.
+
+The field names defined herein should be added as fields in the Item Properties object. 
+
+When specified on an Item, the values are assumed to apply to all Assets in that Item.  For example, an Item may have 
+several related Assets each representing a band or layer for the Item, and which typically all use the same CRS, 
+e.g. a UTM Zone.  However, there may also be assets intended for display, like a preview image or thumbnail, that have 
+been reprojected to a different CRS, e.g., Web Mercator, to better accommodate that use case.  This case of differing 
+projections per Asset is not currently handled by this extension.
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| proj:epsg        | integer\|null   | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:proj4       | string \|null   | Recommended. PROJ4 string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
+| proj:wkt2        | string \|null   | Recommended. WKT2 string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
+| proj:projjson    | PROJJSON Object \|null   | Recommended. PROJJSON object representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
```

Comment:
Linking the "PROJJSON Object" to the PROJJSON readme would be nice.

Reply:
updated